packages/player-component/src/dotlottie-player.ts

```diff
@@ -618,21 +725,252 @@ export class DotLottiePlayer extends LitElement {
+                                        viewBox="0 0 24 24"
+                                        fill="none"
+                                        xmlns="http://www.w3.org/2000/svg"
+                                      >
+                                        <path
+                                          fill-rule="evenodd"
+                                          clip-rule="evenodd"
+                                          d="M20.5281 5.9372C20.821 6.23009 20.821 6.70497 20.5281 6.99786L9.46297 18.063C9.32168 18.2043 9.12985 18.2833 8.93004 18.2826C8.73023 18.2819 8.53895 18.2015 8.39864 18.0593L3.46795 13.0596C3.1771 12.7647 3.1804 12.2898 3.47532 11.999C3.77024 11.7081 4.2451 11.7114 4.53595 12.0063L8.93634 16.4683L19.4675 5.9372C19.7604 5.64431 20.2352 5.64431 20.5281 5.9372Z"
+                                          fill="#20272C"
+                                        />
+                                      </svg>
+                                    `
+                                  : html``}
+                              </div>
+                              <div>${animationName}</div>
+                            </div>
+                          `;
+                        })}
+                      </div> `
+                  : html``}
+                ${this._styleTabIsOpen
+                  ? html`<div
+                        class="option-title-button"
+                        tabindex="0"
+                        aria-label="Back to main popover menu"
+                        @click=${(): void => {
+                          this._styleTabIsOpen = !this._styleTabIsOpen;
+                          this.requestUpdate();
+                        }}
+                      >
+                        <div class="option-title-chevron">
+                          <svg
+                            width="24"
+                            height="24"
+                            viewBox="0 0 24 24"
+                            fill="none"
+                            xmlns="http://www.w3.org/2000/svg"
+                          >
+                            <path
+                              fill-rule="evenodd"
+                              clip-rule="evenodd"
+                              d="M13.5303 6.46967C13.8232 6.76256 13.8232 7.23744 13.5303 7.53033L9.06066 12L13.5303 16.4697C13.8232 16.7626 13.8232 17.2374 13.5303 17.5303C13.2374 17.8232 12.7626 17.8232 12.4697 17.5303L7.46967 12.5303C7.17678 12.2374 7.17678 11.7626 7.46967 11.4697L12.4697 6.46967C12.7626 6.17678 13.2374 6.17678 13.5303 6.46967Z"
+                              fill="#20272C"
+                            />
+                          </svg>
+                        </div>
+                        <div>Styles</div>
+                      </div>
+                      <div class="option-title-separator"></div>
+                      <div class="option-row">
+                        ${this.styles().map((styleName) => {
```

Comment:
let's use the same naming conventions used in dotLottie-js, so from creation to delivery and playing we have consistent naming for dotLottie assets 
```suggestion
                        ${this.themes().map((themeId) => {
```

Reply:
Have rename:

```
                        ${this._themesForCurrentAnimation.map((themeName) => {

```